hr_payslip_ytd_amount/report/report_payslip_ytd_amount.rml

```diff
@@ -0,0 +1,326 @@
+          <para style="P15">[[ translate_term('Reference', o) ]]</para>
+        </td>
+        <td>
+          <para style="P4">[[ o.number or '' ]]</para>
+        </td>
+        <td>
+          <para style="P15">[[ translate_term('Bank Account', o) ]]</para>
+        </td>
+        <td>
+          <para style="P4">[[ o.employee_id.otherid or '' ]]</para>
+        </td>
+      </tr>
+    </blockTable>
+    <blockTable colWidths="63.0,206.0,89.0,181.0" style="Table6">
+      <tr>
+        <td>
+          <para style="P15">Date From</para>
+        </td>
+        <td>
+          <para style="P4">[[ o.date_from or '']]</para>
+        </td>
+        <td>
+          <para style="terp_default_Bold_8">
+            <font face="Helvetica" size="8.0">[[ translate_term('Date To', o) ]]</font>
+          </para>
+        </td>
+        <td>
+          <para style="P4">[[ o.date_to or '' ]]</para>
+        </td>
+      </tr>
+    </blockTable>
+    <para style="P6">
+      <font color="white"> </font>
+    </para>
+    <para style="P6"/>
+    <blockTable colWidths="353,93.0,93.0" style="Table8">
+      <tr>
+        <td>
+          <para style="P13">[[ translate_term('Name', o) ]]</para>
+        </td>
+        <td>
+          <para style="P13">[[ translate_term('Amount', o) ]]</para>
+        </td>
+        <td>
+          <para style="P14">[[ translate_term('Amount Year-to-date', o) ]]</para>
+        </td>
+      </tr>
+    </blockTable>
+    <section>
+      <para style="P4">[[ repeatIn(get_payslip_lines(o.line_ids), 'p') ]]</para>
+      <blockTable colWidths="353,93.0,93.0" style="Table9">
+        <tr>
+          <td>
+            <para style="P4">[[ translate_term(p.name, o, is_payslip_line=True) ]]</para>
+          </td>
+          <td>
+            <para style="P5">[[ formatLang(p.total, currency_obj = o.company_id and o.company_id.currency_id)]]</para>
+          </td>
+          <td>
+            <para style="P5">[[ formatLang(p.total_ytd, currency_obj = o.company_id and o.company_id.currency_id)]]</para>
+          </td>
+        </tr>
+      </blockTable>
+    </section>
+    <para style="P10">
+      <font color="white"> </font>
+    </para>
+    <para style="P16">
+      <font color="white"> </font>
+    </para>
+    <para style="P6">
+      <font color="white"> </font>
+    </para>
+    <para style="P1">
+      <font color="white"> </font>
+    </para>
+    <blockTable colWidths="269.0,269.0" style="Table13">
+      <tr>
+        <td>
+          <para style="P4">
+            <font color="white"> </font>
+          </para>
+        </td>
+        <td>
+          <para style="P12">
+            <font color="white"> </font>
+          </para>
+          <para style="P12">
+            <font color="white"> </font>
+          </para>
+          <para style="P12">
+            <font color="white"> </font>
+          </para>
+        </td>
+      </tr>
+    </blockTable>
+    <para style="P3">
+      <font color="white"> </font>
+    </para>
```

Comment:
Is there a use for all this whitespace? I know RML is a bit horrible, but in case specific positioning is required then
frames can be used. (See check writing in l10n-canada). Otherwise, this seems like useless whitespace.